dove/src/cmd/key.rs

```diff
@@ -0,0 +1,179 @@
+use std::str::FromStr;
+use anyhow::Result;
+use structopt::StructOpt;
+use url::Url;
+use crate::access;
+
+const DEFAULT_NODE_ADDRESS: &str = "ws://127.0.0.1:9944";
+
+/// Secret Key Management
+#[derive(StructOpt, Debug)]
+#[structopt(setting(structopt::clap::AppSettings::ColoredHelp))]
+pub enum Key {
+    /// Save the secret key for access under a alias
+    #[structopt(setting(structopt::clap::AppSettings::ColoredHelp))]
+    #[structopt(name = "add")]
+    Add {
+        /// Alias to access the key. Case-insensitive
+        #[structopt(long)]
+        alias: String,
+
+        /// Access to the key without a password. We do not recommend using this parameter.
+        #[structopt(long = "nopassword")]
+        without_password: bool,
+    },
+
+    /// List of saved keys
+    #[structopt(name = "list")]
+    #[structopt(setting(structopt::clap::AppSettings::ColoredHelp))]
+    List {},
+
+    /// Delete a key
+    #[structopt(name = "delete")]
+    #[structopt(setting(structopt::clap::AppSettings::ColoredHelp))]
+    Delete {
+        /// Delete a key by alias. Case-insensitive
+        #[structopt(long)]
+        alias: Option<String>,
+
+        /// Delete all keys
+        #[structopt(long)]
+        all: bool,
+    },
+}
+
+impl Key {
+    pub fn apply(&mut self) -> Result<()>
+    where
```

Comment:
I think `Sized` constraint is not needed here, as it was specifically for `dyn Cmd` when it existed?